pm2.json

```diff
@@ -0,0 +1,10 @@
+{
+  "apps": [
+    {
+      "name": "server",
+      "script": "lib/indexApi.js",
+      "error_file": "logs/server/error.log",
+      "out_file": "logs/server/out.log"
+    }
+  ]
+}
```

Comment:
The server should have a real name, and the output should continue going to /var/log:

```json
{
  "apps": [
    {
      "name": "logsServer",
      "script": "./lib/indexApi.js",
      "out_file": "/var/log/logsServer.log"
    }
  ]
}
```

Reply:
Actually, on the production box it's `/var/log/logsApi.log`

Reply:
Also, we should add this to the readme:

### Manage server using `pm2`

First, install pm2 to run at startup:

```sh
yarn global add pm2
pm2 startup # Then do what it says
```

Next, tell pm2 how to run the server script:

```sh
# install:
pm2 start pm2.json
pm2 save

# check status:
pm2 monit
tail -f /var/log/logsServer.log

# manage:
pm2 restart logsServer
pm2 reload logsServer
pm2 stop logsServer
```